tests/test_insert_deletes_consolidate.cpp

```diff
@@ -79,59 +75,115 @@ inline void load_aligned_bin_part(const std::string& bin_file, T* data,
 }
 
 std::string get_save_filename(const std::string& save_path,
-                              size_t points_to_skip, size_t points_deleted,
+                              size_t points_to_skip, float points_deleted,
```

Comment:
Any reason to change the type from size_t to float?

Reply:
Because points_deleted is set to the percentage of deleted points